QLogger.h

```diff
@@ -43,10 +43,13 @@ class QLoggerManager
     * @brief Gets an instance to the QLoggerManager.
     * @return A pointer to the instance.
     */
-   static QLoggerManager *getInstance();
+   static QLoggerManager &getInstance();
+
+   bool hasModules() const;
+   bool hasModule(const QString &module) const;
```

Comment:
Would be nice to have documentation as the other methods.

Reply:
Yes, that's possible but I  think these methods are already self explanatory.

Reply:
Yeah, they are, but they should look the same as the rest, documentation included.